domains/bimarsha.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Bimarsha69",
+           "email": "116154292+Bimarsha69@users.noreply.github.com",
+           "discord": "1182856767286288507"
+        },
+    
+        "record": {
+            "TXT": "ad9b8ff5793eb8dd58746a30bee4c7"
+            "HOSTNAME":"_github-pages-challenge-Bimarsha69.bimarsha"
```

Comment:
you'd need to make another .json file if you want to make a github pages verification domain